instances/latest/terminologies/technique/nearInfraredSpectoscropy.jsonld

```diff
@@ -0,0 +1,14 @@
+{
```

Comment:
typo in "spectroscopy" (you wrote: spectoscropy)

fix in:
file name
at_id
name

Reply:
done